stm32-modules/heater-shaker/tests/test_gcodes.cpp

```diff
@@ -1,7 +1,10 @@
 #include <array>
 
 #include "catch2/catch.hpp"
+#pragma GCC diagnostic push
```

Comment:
It would be nice to split this file up, yes. But I don't think it needs to be in this PR.